skipchain/api.go

```diff
@@ -201,6 +203,9 @@ func (c *Client) GetUpdateChain(roster *onet.Roster, latest SkipBlockID) (reply
 			if err == nil && len(r2.Update) != 0 {
 				break
 			}
+			time.Sleep(delay)
+			// exponential backoff
+			delay *= 2
```

Comment:
this will make the last retry very long without doing anything, can you an if condition when `i == retries - 1`?